src/full/Agda/TypeChecking/Errors.hs

```diff
@@ -1566,6 +1608,9 @@ instance PrettyTCM TypeError where
         _ -> prettyTCM t <+> "is not usable at the required modality"
          <+> pure (attributesForModality mod)
 
+    InvalidFieldModality coh -> fsep $
+      pwords "Cannot have record fields with modality " ++ [pretty coh]
```

Comment:
The trailing space is unnecessary here.  I suppose it will be just swallowed by `pwords`, but I am not 100% sure.